### PR TITLE
Add command to check if a website is up or not

### PR DESCRIPTION
**Pull request by jesse-c**
## Description

Add command to check if a website is up or not. It uses the isitup.org service.

## Type of change

- [x] New script command

## Screenshot

image

## Dependencies / Requirements

N/A

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/is-it-up.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```
Comment (thomaspaulmann) [inline commands/developer-utils/is-it-up.sh:2]: Let's add a hint that this script uses a dependency as described in the contribution guidelines even though `jq` is a rather common one.
```suggestion

# Dependency: This script requires `jq` cli installed: https://stedolan.github.io/jq/
# Install via homebrew: `brew install jq`

```

File: commands/developer-utils/is-it-up.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Is It Up?
+# @raycast.mode compact
```
Comment (thomaspaulmann) [inline commands/developer-utils/is-it-up.sh:6]: Could also be neat to have as an `inline` script for specific websites that you want to keep observing.

File: commands/developer-utils/is-it-up.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+# @raycast.packageName Developer Utils
+#
+# Optional parameters:
+# @raycast.icon 🩺
+# @raycast.currentDirectoryPath ~
+# @raycast.needsConfirmation false
```
Comment (thomaspaulmann) [inline commands/developer-utils/is-it-up.sh:12]: The default value is `false`. You can remove the parameter.

```suggestion
```

File: commands/developer-utils/is-it-up.sh [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Is It Up?
+# @raycast.mode compact
+# @raycast.packageName Developer Utils
+#
+# Optional parameters:
+# @raycast.icon 🩺
+# @raycast.currentDirectoryPath ~
```
Comment (thomaspaulmann) [inline commands/developer-utils/is-it-up.sh:11]: Is this necessary? I assume that the script works from every directory?

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Script looks good. I've added some inline comments, nothing major. Most importantly, is it necessary to execute the script from `~`? This could be different on user machines but I also don't see anything that would prevent somebody to execute it from somewhere else. If not, I would remove the `currentWorkingDirectory` parameter.

File: commands/developer-utils/is-it-up.sh (hunk 3 above)
Comment (jesse-c) [reply commands/developer-utils/is-it-up.sh]: Done.

File: commands/developer-utils/is-it-up.sh (hunk 4 above)
Comment (jesse-c) [reply commands/developer-utils/is-it-up.sh]: It would! I've removed it.

File: commands/developer-utils/is-it-up.sh (hunk 2 above)
Comment (jesse-c) [reply commands/developer-utils/is-it-up.sh:9]: It would! I guess people could copy the script and remove the argument?

Comment (thomaspaulmann) [reply commands/developer-utils/is-it-up.sh:9]: Yeah, let them be creative 🎨

**Pull request merged**